x-pack/plugins/fleet/server/routes/epm/index.ts

```diff
@@ -74,7 +74,7 @@ import { getFileHandler } from './file_handler';
 const MAX_FILE_SIZE_BYTES = 104857600; // 100MB
 
 export const INSTALL_PACKAGES_AUTHZ: FleetAuthzRouteConfig['fleetAuthz'] = {
-  integrations: { installPackages: true },
+  integrations: { installPackageAssets: true },
 };
```

Comment:
Fleet team: This change decouples integration installation from fleet privileges.  (See #183020)

Package Service methods impacted:

- `ensureInstalledPackage`
- `installPackage`
- `installCustomIntegration`
- `reinstallEsAssets`

REST endpoints impacted:

- `EPM_API_ROUTES.INSTALL_FROM_REGISTRY_PATTERN`
- `EPM_API_ROUTES.CUSTOM_INTEGRATIONS_PATTERN`
- `EPM_API_ROUTES.INSTALL_FROM_REGISTRY_PATTERN_DEPRECATED`
- `EPM_API_ROUTES.REAUTHORIZE_TRANSFORMS`

Reply:
Unfortunately not, since the existing key is used in the UI to toggle certain elements so removing it would break that. 

It's not a matter of simply swapping them out either since the UI has been built with the assumption that writeIntegrationPolicies && integrations.all are coupled.